app/src/main/java/com/example/mymovie/MovieFragment.java

```diff
@@ -0,0 +1,90 @@
+package com.example.mymovie;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+
+public class MovieFragment extends Fragment {
+
+    private ImageView image;
+    private TextView name, detail;
+    private Button detailBtn;
+    private int resId;
+    private String movieName, rate;
+
+    public final static String MOVIE = "movie";
```

Comment:
MOVIE 라는 변수명만을 가지고는 용도를 알 수 없습니다. 명확한 변수명을 사용해 주세요.

```suggestion
    public final static String BUNDEL_MOVIE = "movie";
```